guides/delivery/setting-up-a-new-project.md

```diff
@@ -61,28 +61,66 @@ Before the project starts there’s also a number of tasks, mostly administrativ
 
 ## Kick off meeting
 
-While the project is being set up, and before it starts, a kick off meeting should happen. The purpose of this is to introduce the team to each other, discuss and agree how the team want to work, and to create a plan for the first few days of the project.
+While the project is being set up, and before it starts, a kick off should happen. This ensures:
+- The team get introduced to each other & discuss and agree how they want to work
+- The team and client know and understand what the work is all about
+- We realise our priorities so we have an initial plan
+- We discuss ways of working
 
-You should have everyone working on the project in this meeting (even if they’re part-time).
-
-The meeting can be structured in any way you want, but there’s a number of different outcomes you should look to get out of it.
+You should have everyone working on the project in this meeting (even if they’re part-time). 
 
 ### Introductions
 
-Use the kick-off to introduce everyone to each other. It shouldn’t just be about people’s names, but roles and what each person will be doing on the project. Highlight people’s experience and expertise, particularly if it’s going to benefit the project or subject matter you’re working on.
+Use the kick-off to introduce everyone to each other. It might be a good idea to give team members a bit of advance notice about this, so that they can already think and prepare their own intros tailored to this project.
+
+It shouldn’t just be about people’s names, but roles and what each person will be doing on the project. Some further ideas:
+- Highlight people’s experience and expertise, particularly if it’s going to benefit the project or subject matter you’re working on
+- It is also important to say if any of the team members have worked on a previous phase of the same project/product/service
+make sure to point out who is full time/part time or if there is any different working pattern (such as compressed hours, any days they are not working, etc…)
+- Prep the client what the day will look like and set expectations. For example who the client should bring and what the agenda should be.
+
+### Get to a shared understanding of what the work and priorities are
+There are a number of outcomes you should look to get out of a kick-off. This is important especially if we are working with that client for the first time and there are areas that are not very clear to us, or to them. You can structure a kick-off in the way that works best for you, but some frameworks that will help you get the outcomes you need are below:
+- [Good questions to kick off a discovery] (https://www.dxw.com/2019/06/good-questions-to-kick-off-a-discovery/)
```

Comment:
There should be no space between the closing square bracket and the opening round bracket.